client/js/components/map/admin/AdminLayerListItem.vue

```diff
@@ -30,119 +30,118 @@
-        </span>
-        <span
-          class="font-size-smaller u-mr-0_5"
-          v-if="layer.attributes.layerWithChildrenHidden">
+        <br>{{ Translator.trans('maplayer.hidden.child.of.category') }}
+      </span>
+      <span
+        class="font-size-smaller u-mr-0_5"
+        v-if="layer.attributes.layerWithChildrenHidden">
           <!-- categories that should appear as Layer -->
-          <br>{{ Translator.trans('maplayer.category.with.hidden.children') }}
-        </span>
-        <span
-          class="font-size-smaller u-mr-0_5"
-          v-if="layer.attributes.description">
-          <br>{{ layer.attributes.description }}
-        </span>
-        <span
-          class="font-size-smaller u-mr-0_5"
-          v-if="layer.attributes.isBplan">
-          <br>{{ Translator.trans('explanation.gislayer.useas.bplan') }}
-        </span>
-        <span
-          class="font-size-smaller u-mr-0_5"
-          v-if="layer.attributes.isScope">
-          <br>{{ Translator.trans('explanation.gislayer.useas.scope') }}
-        </span>
-        <span
-          class="font-size-smaller"
-          v-if="false === layer.attributes.isEnabled">
-          <br>{{ Translator.trans('explanation.gislayer.useas.invisible') }}
-        </span>
-        <span
-          class="font-size-smaller"
-          v-if="layer.attributes.isPrint">
-          <br>{{ Translator.trans('explanation.gislayer.useas.print') }}
-        </span>
-      </div>
-      <!--
-        Show this Stuff (Visibility-group / show initially on load) only for layer, not for Categories
-      -->
-      <template v-if="(layer.type === 'GisLayer') && hasPermission('feature_map_layer_visibility')">
-        <div class="w-1/12 text-right">
-          <a
-            v-if="('undefined' !== typeof activeLayer.id || '' !== hoverLayerId) && false === layer.attributes.isBaseLayer && (false === isChildOfCategoryThatAppearsAsLayer)"
-            @click.stop.prevent="toggleVisibilityGroup"
-            @mouseover="setIconHoverState"
-            @mouseout="unsetIconHoverState"
-            :title="hintTextForLockedLayer">
-            <i :class="[iconClass,showGroupableIcon]" />
-          </a>
-        </div>
-        <div class="w-1/12 text-right">
-          <input
-            type="checkbox"
-            data-cy="adminLayerListItem:toggleDefaultVisibility"
-            :disabled="'' !== layer.attributes.visibilityGroupId || (true === isChildOfCategoryThatAppearsAsLayer)"
-            @change.prevent="toggleHasDefaultVisibility"
-            :checked="hasDefaultVisibility"
-            :class="iconClass">
-        </div>
-      </template>
-      <!--
-        Show this Stuff for 'special category that looks like an Layer and hides all his children'
-      -->
-      <div
-        v-if="(layer.type === 'GisLayerCategory' && layer.attributes.layerWithChildrenHidden)"
-        class="w-2/12 text-right">
+        <br>{{ Translator.trans('maplayer.category.with.hidden.children') }}
```

Comment:
```suggestion
       {{ Translator.trans('maplayer.category.with.hidden.children') }}
```
This and following: I think we could remove the `<br>` tag and solve this on css level instead. https://developer.mozilla.org/en-US/docs/Web/CSS/line-height Or is there a particular reason for the` <br>`-tag?

Reply:
I'm ok with that :+1: